docs/book/src/tasks/experimental-features/experimental-features.md

```diff
@@ -41,17 +41,16 @@ Another way is to set them as environmental variables before running e2e tests.
 On development environments started with `Tilt`, features can be enabled by setting the feature variables in `kustomize_substitutions`, e.g.:
 
 ```yaml
-{
-  "enable_providers": ["kubeadm-bootstrap","kubeadm-control-plane"],
-  "allowed_contexts": ["kind-kind"],
-  "default_registry": "gcr.io/cluster-api-provider",
-  "provider_repos": [],
-  "kustomize_substitutions": {
-    "EXP_CLUSTER_RESOURCE_SET": "true",
-    "EXP_MACHINE_POOL": "true",
-    "CLUSTER_TOPOLOGY": "true"
-  }
-}
+enable_providers:
+- docker
+- kubeadm-bootstrap
+- kubeadm-control-plane
+allowed_contexts:
+- kind-capi-test
```

Comment:
What about dropping everything not relevant to the current topic (so we don't have to maintain it)
```suggestion
...
```

Reply:
Done